utils/src/main/kotlin/Ci.kt

```diff
@@ -24,5 +24,8 @@ package org.ossreviewtoolkit.utils
  */
 object Ci {
     val isAppVeyor = listOf("APPVEYOR", "CI").all { Os.env[it]?.toBoolean() == true }
+    val isAzure = Os.env["AGENT_OS"] != null
```

Comment:
[This page](https://docs.microsoft.com/en-us/azure/devops/pipelines/migrate/from-travis?view=azure-devops#predefined-variables) the proper equivalent would be to check the `TF_BUILD` environment variable to be `True`.

Reply:
Done.